### ISSUE TITLE
Incompatible with fastify 1.0-rc.1

### ISSUE THREAD

**Issue by yamalight** [fastify/fastify-auth#9]
fastify-auth is currently incompatible with 1.0-rc.1, throws the following error:
```js
    fastify-plugin - expected '>=0.13.1' fastify version, '1.0.0-rc.1' is installed

      1 | // npm packages
      2 | const initFastify = require('fastify');
    > 3 | const fastifyAuth = require('fastify-auth');
```

Updating fastify-plugin should probably fix this, right? Should I send a PR?

**Comment by mcollina** [comment]
Unfortunately that commit didn't really fix it. I'll need to get some more work done.

**Comment by yamalight** [comment]
@mcollina does [this comment](https://github.com/fastify/fastify/issues/229#issuecomment-364937987) covers all that needs to be done? If that's the case - I can prepare a PR in a few mins

**Comment by mcollina** [comment]
there is a little bit more work involved, as tests are not currently
passing.
Il giorno lun 12 feb 2018 alle 09:41 Tim Ermilov <notifications@github.com>
ha scritto:

**Comment by jsumners** [comment]
The tests are written against an outdated pattern. They are attempting to register routes _after_ the Fastify server is already listening. The tests themselves should be fine, there just needs to be a bit of refactoring done to build the test server correctly, e.g. https://github.com/fastify/fastify-url-data/commit/65b158bf46a36881f9756c08d46764911dfaff74.

**Comment by mcollina** [comment]
the tests depends on fastify-leveldb being updated as well. I've already done the work, I just need publish rights (but @delvedor is skying somewhere in the mountains).

**Comment by mcollina** [comment]
Solved in 0.2.0.
